ansible/roles/stig/tasks/TOSS_04_010020.yml

```diff
@@ -10,7 +10,29 @@
 
 
 - name: TOSS-04-010020 - TOSS, for PKI-based authentication, must enforce authorized access to the corresponding private key.
-  # TODO: Implement the task
+  block:
+    - name: Find all private ssh key files
+      ansible.builtin.command: 'find / -xdev -type f -regex ".*/id_[a-z0-9]*[^\.pub]$"'
```

Comment:
I didn't actually know about `-regex`, that's cool! -- I've always done variations on `-name`